symfony/config/packages/twig.php

```diff
@@ -10,7 +10,7 @@
         'isStable' => $env->isStable(),
         'reportKitUrl' => $env->configuration['reportKitUrl'] ?? '',
         'assetVer' => $env->values['assetVer'],
-        'awardeeOrganization' => '',
+        'site' => '@App\Service\SiteService',
```

Comment:
I'm using this in my branch, so I know it works. I don't _think_ we'll ever run the risk of a name collision (e.g. say we were iterating through a list of `sites` and called the iterator `site`, wiping out the service in the process).